example/cluster.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const cluster = require('cluster');
+const express = require('express');
+const metricsServer = express();
+const AggregatorRegistry = require('../').AggregatorRegistry;
+
+if (cluster.isMaster) {
+	for (let i = 0; i < 4; i++) {
+		cluster.fork();
+	}
+
+	metricsServer.get('/cluster_metrics', (req, res) => {
+		AggregatorRegistry.clusterMetrics((err, metrics) => {
```

Comment:
`clusterMetrics` is not a static method and thus can only be called on an Instance - of which there is none in your example code